README.md

```diff
@@ -86,6 +86,34 @@ The cache uses `catbox` to provide a simple pluggable interface, this supports s
 * http-transport:{version}:response - Basic response from a call cached for the duration of the `max-age` value key on just the URL of the response.
 * http-transport:{version}:staleResponse - Stale response from a called cached for the `stale-if-error` value keyed on just the URL of the response.
 
+Additionally, cache keys can be configured by passing a `varyOn` option. `varyOn` should be an array containing values that you want to vary on. For example, let's say you make requests to the same endpoint but values of some headers vary. By letting `http-transport-cache` know which headers vary, it will first check whether the headers actually exist in the request. If they do then it will construct a unique cache key which contains headers and their values.
```

Comment:
Could be reworded slightly to make it clearer that the values in the varyOn option are header names
```suggestion
Additionally, cache keys can be configured by passing a `varyOn` option. `varyOn` should be an array of header names which the cache should additionally vary on; for some use-cases, requests made to the same endpoint but with differing values for certain headers illicit different responses - and therefore cannot share the same cached response e.g.`accept-language`. By letting `http-transport-cache` know which headers to vary on, a unique cache key will be constructed which also contains said headers and their values.
```